src/ipld/cid_hashset.rs

```diff
@@ -2,27 +2,84 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use ahash::HashSet;
-use cid::Cid;
+use cid::{Cid, Version};
+use fvm_ipld_encoding::DAG_CBOR;
+use std::collections::BTreeSet;
 
+const BLAKE2B256: u64 = 0xb220;
+
+// Nearly all Filecoin CIDs are V1, DagCbor encoded, and hashed with Blake2b265. If other types of
+// CID become popular, they can be added to the CidVariant structure.
+enum CidVariant {
+    V1DagCborBlake2b([u8; 32]),
```

Comment:
`32` could be a named constant given that it appears a few times. Alternatively, the entire `[u8; 32]` could be aliased.

Reply:
Hoisted out the size of the blake2b hash.